src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/ImageLoader.java

```diff
@@ -0,0 +1,171 @@
+package com.microsoft.maui;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.widget.ImageView;
+
+import androidx.annotation.ColorInt;
+import androidx.annotation.Nullable;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.RequestBuilder;
+import com.bumptech.glide.RequestManager;
+import com.bumptech.glide.load.DataSource;
+import com.bumptech.glide.load.engine.DiskCacheStrategy;
+import com.bumptech.glide.load.engine.GlideException;
+import com.bumptech.glide.request.RequestListener;
+import com.bumptech.glide.request.target.Target;
+import com.microsoft.maui.glide.fontimagesource.FontModel;
+
+import java.io.File;
+import java.io.InputStream;
+
+public class ImageLoader {
```

Comment:
One last thought on this...

For every Java class, we have to make a JNI `FindClass()` call. So should we move all the methods here into `ViewHelper.java`?

Then maybe `ViewHelper` isn't a generic enough name, maybe something like `JavaBridge` would make sense if we put every single method inside?